File: pgm/ABC_R/ABC_194_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
solid15 <- 15
solid10 <- 10
solid3 <- 3
fat8 <- 8
fat3 <- 3
one <- 1
two <- 2
three <- 3
four <- 4
space <- " "
blank <- ""
A <- strsplit(arr[one], split = space)[[one]][one] |>
    as.integer()
B <- strsplit(arr[one], split = space)[[one]][two] |>
    as.integer()
AB <- A + B
if (solid15 <= AB && fat8 <= B) {
    one |>
        cat("\n", sep = blank)
} else if (solid10 <= AB && fat3 <= B) {
    two |>
        cat("\n", sep = blank)
} else if (solid3 <= AB) {
    three |>
        cat("\n", sep = blank)
} else {
    four |>
        cat("\n", sep = blank)
}
q("no")

File: pgm/ABC_Dart/ABC_194_A.dart
// import 'dart:collection';
// import 'dart:convert';
import 'dart:io';
// import 'dart:math';
// import 'dart:typed_data';

void main() {
  final [int A, int B] = stdin
      .readLineSync()!
      .split(' ')
      .map(int.parse)
      .toList();
  const int solid15 = 15;
  const int solid10 = 10;
  const int solid3 = 3;
  const int fat8 = 8;
  const int fat3 = 3;
  const int one = 1;
  const int two = 2;
  const int three = 3;
  const int four = 4;
  final int AB = A + B;
  print(
    solid15 <= AB && fat8 <= B
        ? one
        : solid10 <= AB && fat3 <= B
        ? two
        : solid3 <= AB
        ? three
        : four,
  );
  exit(0);
}
